clear-all-rate-limits/clear_all_rate_limits.py

```diff
@@ -23,16 +23,17 @@
 num_errors = 0
 
 
-
 def main():
     account_access_token = sys.argv[1]
```

Comment:
Nit: `argv` length isn't checked prior to access, leading to an exception if the correct number of arguments aren't passed.

Reply:
Yes, the same thing happens in the other examples but that's okay. The intention of this repository is to demonstrate our API, not to provide production-ready scripts.